Library/Homebrew/vendor/README.md

```diff
@@ -0,0 +1,56 @@
+Vendored Dependencies
+=====================
+
+* okjson is a pure-Ruby JSON parser written by
+[Keith Rarick](https://github.com/kr). Its upstream repository can be found
+[here](https://github.com/kr/okjson).
+
+* ruby-macho is a pure-Ruby Mach-O parser written by
+[William Woodruff](https://github.com/woodruffw). Its upstream repository can
+be found [here](https://github.com/woodruffw/ruby-macho).
```

Comment:
I'm happy with the existing language if Mike is. My thoughts were just to include a version or even commit reference alongside the current language just to be clear at what point it was vendored from.